Rakefile

```diff
@@ -0,0 +1,13 @@
+# exit gracefully
+trap('SIGINT') { exit(0) }
+trap('SIGTERM') { exit(0) }
+
+task :run do
+  ruby 'main.rb'
+end
+
+task :shell do
+  sh 'bash'
```

Comment:
What's this line do?

Reply:
Because I changed the Dockerfile from `CMD` to an `ENTRYPOINT` to rake, I created this to easily be able to run a bash shell by running `docker-compose run --rm bash`.